modules/prometheus-cpp/1.3.0/source.json

```diff
@@ -0,0 +1,5 @@
+{
+    "url": "https://github.com/jupp0r/prometheus-cpp/releases/download/v1.3.0/prometheus-cpp-with-submodules.tar.gz",
```

Comment:
With Bazel the vanilla tarball (generated from pure git repo) should be sufficient because Bazel takes care of dependencies like civetweb or Google Test on its own.

https://github.com/jupp0r/prometheus-cpp/archive/refs/tags/v1.3.0.tar.gz

Reply:
Such URL may not always be stable, see https://blog.bazel.build/2023/02/15/github-archive-checksum.html